pkg/cmd/grafana-cli/services/api_client.go

```diff
@@ -106,6 +126,37 @@ func (client *GrafanaComClient) ListAllPlugins(repoUrl string) (models.PluginRep
 }
 
 func sendRequest(client http.Client, repoUrl string, subPaths ...string) ([]byte, error) {
+	req, err := createRequest(repoUrl, subPaths...)
```

Comment:
Can you replace this with a call to `executeRequestReader`, and use the returned `ReadCloser`? Seems like you're repeating a lot of the same logic.

Reply:
You're right, I've refactored that and renamed these two methods.